Makefile

```diff
@@ -494,6 +494,11 @@ test: generate fmt vet manifests build-cli-mocks ## Run tests
 	$(MAKE) kubebuilder -C $(TOOLS_DIR)
 	KUBEBUILDER_ASSETS=$(ROOT_DIR)/$(KUBEBUILDER)/bin $(MAKE) test -C addons
 
+	# pinniped post-deploy
```

Comment:
question: these tests are already invokes as a part of `make test -C addons`, do we need to invoke them again?

Reply:
I am removing it from addons. Reasoning being faster turn around for testing just addons module.